src/redux/data.js

```
@@ -830,6 +834,8 @@ export const dataWatchPendingTransactions = (
               type: tx?.type,
             });
             updatedPending.status = newStatus;
+          } else if (currentNonce > tx.nonce) {
+            updatedPending.status = TransactionStatusTypes.unknown;
```

should we throw a custom error to sentry to know how often this happens?